worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeBiomeRegistry.java

```diff
@@ -72,12 +72,13 @@ static void populate() {
         Map<Integer, BiomeGenBase> biomes = HashBiMap.create();
         Map<Integer, BiomeData> biomeData = new HashMap<Integer, BiomeData>();
 
-        for (BiomeGenBase biome : BiomeGenBase.getBiomeGenArray()) {
+        for (BiomeGenBase biome : BiomeGenBase.biomeRegistry) {
             if ((biome == null) || (biomes.containsValue(biome))) {
                 continue;
             }
-            biomes.put(biome.biomeID, biome);
-            biomeData.put(biome.biomeID, new ForgeBiomeData(biome));
+            int biomeId = BiomeGenBase.getIdForBiome(biome);
```

Comment:
iirc this was added because biomes didn't have a registry in minecraft, so doing a biome operation could cause us to check through a hundred biomes for every block we wanted to change, especially with modpacks that add a lot of biomes.
since the internal registry now maps them for us, it shouldn't be too much of an issue. (is the internal registry a bimap? i know forge uses bimaps for some of the other registries at least, but don't know if the "nms" one is.)